common/zarf.yaml

```diff
@@ -34,8 +34,8 @@ components:
             wait:
               cluster:
                 kind: packages.uds.dev
-                name: postgres
-                namespace: postgres
+                name: postgres-operator
```

Comment:
So there actually is a CR in both. I am fine with checking the operator but this might unintentionally be a "breaking change" if the cluster is not up and it causes an app that uses this as a dependency to fail. Granted, the postgres namespace is technically optional albeit how we are deploying in the wild. 

t would be nice to actually check both, but there isnt a way to optionally check the cr in the postgres namespace 😢

Reply:
@cmharden can you add an onDeploy.after cmd that waits for for the package cr status in the postgres namespace if it exists? That would fix what you needed an alleviate my concern 🙏

Reply:
Logic added